source/extensions/filters/network/thrift_proxy/router/router_impl.h

```diff
@@ -188,6 +201,11 @@ class Router : public Tcp::ConnectionPool::UpstreamCallbacks,
   void setDecoderFilterCallbacks(ThriftFilters::DecoderFilterCallbacks& callbacks) override;
   bool passthroughSupported() const override { return passthrough_supported_; }
 
+  // Stats
+  void incCounter(Stats::Scope& scope, Stats::StatName name) {
+    scope.counterFromStatName(name).inc();
```

Comment:
should we just get the scope via `cluster_->statsScope()` instead of passing it around?